js/network.js

```diff
@@ -0,0 +1,37 @@
+const getData = (onSuccess, onFail) => {
+  fetch('https://27.javascript.pages.academy/kekstagram-simple/data')
+    .then((response) => {
+      if (response.ok) {
+        return response.json();
+      }
+      throw new Error(`${response.status} — ${response.statusText}`);
+    })
+    .then((data) => onSuccess(data))
```

Comment:
Если из колдбка переменная передаётся в функцию, можно так
```suggestion
    .then(onSuccess)
```

И в других подобных местах

Reply:
Сделал